lanes/mev/check_tx.go

```diff
@@ -63,17 +63,17 @@ type (
 	}
 )
 
-// NewCheckTxHandler is a constructor for CheckTxHandler.
+// NewCheckTxHandler constructs a new CheckTxHandler instance.
 func NewCheckTxHandler(
 	baseApp BaseApp,
 	txDecoder sdk.TxDecoder,
-	tobLane TOBLaneI,
+	mevLane MEVLaneI,
```

Comment:
Have we agreed on `mevLane` for the TOB lane? I'd prefer smth more descriptive i.e `auctionLane` or similar. Otherwise, happy to keep the same